packages/jetpack-options/tests/Manager.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+use WP_Mock\Tools\TestCase;
+use Jetpack\V7\Options\Manager;
+
+class ManagerTest extends TestCase {
+	public function setUp(): void {
+		\WP_Mock::setUp();
+
+		$this->manager = new Manager_Test();
+	}
+
+	function test_get_private_option_returns_value() {
+		\WP_Mock::userFunction( 'get_option', array(
+			'times' => 1,
+			'args' => array( 'jetpack_private_options' ),
+			'return' => array( 'private_name' => true ),
+		) );
+
+		$value = $this->manager->get_option( 'private_name' );
+
+		// Did Jetpack_Options::get_option() properly return true?
+		$this->assertTrue( $value );
+	}
+
+	function test_get_network_option_returns_value() {
+		\WP_Mock::userFunction( 'get_site_option', array(
+			'times' => 1,
+			'args' => array( 'jetpack_network_name', false ),
+			'return' => true,
+		) );
+		\WP_Mock::userFunction( 'is_multisite', array(
+			'times' => 1,
+			'args' => array(),
+			'return' => true,
+		) );
+
+		$value = $this->manager->get_option( 'network_name' );
+
+		// Did Jetpack_Options::get_option() properly return true?
+		$this->assertTrue( $value );
+	}
+
+	function test_get_non_compact_option_returns_value() {
+		\WP_Mock::userFunction( 'get_option', array(
+			'times' => 1,
+			'args' => array( 'jetpack_uncompact_option_name', false ),
+			'return' => true,
+		) );
+		\WP_Mock::userFunction( 'is_multisite', array(
```

Comment:
Instead of setting this us so many times can we just set it once and then overwrite it when we need to?

Reply:
Yes, I need to make it more DRY, thanks!